### PR TITLE
Added reactotron

### PR DESCRIPTION
**Pull request by nazabalm**

### CODE REVIEW

**Comment by seedwalk** [comment]
You added `.js` files, those shouldn't be `.ts` instead?

**Review by seedwalk — CHANGES_REQUESTED** [review]
You added all `.js` files. Shouldn't those be `.ts`?

File: store.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+/*
```
Comment (seedwalk) [inline store.js:1]: could we put this file inside `src/store` folder?

so it can be used from web app as well?

File: ReactotronConfig.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import Reactotron from 'reactotron-react-native';
```
Comment (seedwalk) [inline ReactotronConfig.js:1]: maybe putting this one inside `src/store`

**Review by seedwalk — APPROVED** [review]

**Review by naniverdier — APPROVED** [review]

**Pull request merged**